lib/utils/types.js

```diff
@@ -40,6 +41,31 @@ module.exports = {
   isVariableDeclarator,
 };
 
+/**
+ * Check whether or not a node is a ClassProperty, with at least the given decorator
```

Comment:
I don't think this needs to be limited to `ClassProperty`.

Reply:
just a comment copy/paste goof. I don't think `hasDecorator` needs to care about what type the node is